compat/src/main/scala-2.11_2.12/scala/collection/compat/PackageShared.scala

```diff
@@ -30,6 +30,8 @@ private[compat] trait PackageShared {
    */
   type Factory[-A, +C] = CanBuildFrom[Nothing, A, C]
 
+  type IsTraversableLikeAux[A1, Repr] = IsTraversableLike[Repr] { type A = A1 }
```

Comment:
You don’t seem to be using this type alias anymore.

Reply:
good catch!